features/kinderen.feature

```diff
@@ -10,7 +10,7 @@ Functionaliteit: Kinderen van een persoon raadplegen
 
     @gba
     Scenario: de naam van een kind is gecorrigeerd
-      Gegeven de persoon met burgerservicenummer '999996150' heeft een 'kind' met de volgende gegevens
+      Gegeven de persoon met burgerservicenummer '000001234' heeft een 'kind' met de volgende gegevens
```

Comment:
wat is dit voor bsn? (zelfde geldt voor de andere ban's in de feature) We zouden toch burgerservicenummers gebruiken uit https://www.rvig.nl/binaries/rvig/documenten/richtlijnen/2018/09/20/test-burgerservicenummers-bsn-en-a-nummers-inclusief-omnummertabel/20201126_Testnummers_%28inclusief_omnummertabel%29_GBA-V.ods

Reply:
Is aangepast.